### ISSUE TITLE
Is it possible to copy scripts to the top level in package.json?

### ISSUE THREAD

**Issue by janhoogeveen** [NYTimes/kyt#470]
I'm trying to integrate prettier [in our starter-kyt](https://github.com/cleverfranke/cf-kyt-starter-universal-redux/compare/feature/prettier).

The `"lint-staged"` key should be copied to the top level part of `package.json` so the key is a sibling of `scripts`, `devDependencies` and `dependencies`. Problem is I only seem to be able to copy this script within the scripts object in package.json which unfortunately doesn't seem to work when running the precommit hook with `husky`.

_Tldr; can we copy parts of our custom package.json file to the root of the generated package.json file? Without that option something like https://github.com/typicode/husky is not useable.

**Comment by janhoogeveen** [comment]
Also, if you think adding Prettier in combination with precommit hooks is a good idea as proposed in #445 I would also be willing to do a PR and get this in kyt-core. Just let me know.

**Comment by delambo** [comment]
I'm not sure if Prettier fits in kyt-core, but I would like to see it in the starter kyts. We have a `kyt` configuration namespace for starter kyt packages. I think we could add another configuration for of items to copy from the starter kyt into a user's project package.json.

If you want to create a pull I'd love to see it.

**Comment by janhoogeveen** [comment]
Sure, I've got a bunch of changes ready to go already. The only thing preventing me from finishing it up is the root level scripts problem. Do you want me to change the starter scripts to account for the extra options or do you want to tackle that internally?

**Comment by janhoogeveen** [comment]
Not needed anymore since Prettier integration has already landed in some form now.
